### PR TITLE
Add apps for hub chains, add sFUEL checks

### LINKED ISSUE #150: Wrapped sFUEL - Max Results in Native Insufficient Value
**Describe the bug**
As sFUEL is the native token and is required to facilitate any transaction, when using wSFUEL in Metaport, if a user selects **MAX** and no remainder is left, then they will not be able to actually transfer the full MAX as they won't have enough sFUEL to do so.

**Expected behavior**
A little bit of sFUEL can go a fairly long way. Expected behavior is that the Maximum value should send all but some X value leaving the user enough to transact a number of times on the SKALE Chain.

### PR DESCRIPTION
**Pull request by dmytrotkk**
resolves https://github.com/skalenetwork/metaport/issues/152
resolves https://github.com/skalenetwork/metaport/issues/150

### CODE REVIEW

File: src/components/TokenList/TokenList.scss [hunk 1]
```diff
@@ -1,11 +1,8 @@
 
 
 .mp__iconToken {
-    width: 21px;
-    height: 21px;
-    // opacity: 0.85;
```
Comment (TheGreatAxios) [inline src/components/TokenList/TokenList.scss:6]: Suggest to Remove Commented CSS

File: src/components/SFuelBadge/SFuelBadge.tsx [hunk 2]
```diff
@@ -7,7 +7,6 @@ import Tooltip from '@mui/material/Tooltip';
 import localStyles from './SFuelBadge.scss';
 
 
-
 const BadgeStates = {
   success: {
     tooltip: 'Ok text',
```
Comment (TheGreatAxios) [inline src/components/SFuelBadge/SFuelBadge.tsx]: Note this says "Ok text" - should it say this?

**Review by TheGreatAxios — APPROVED** [review]
Changes are suggestions

File: src/components/SFuelBadge/SFuelBadge.tsx (hunk 2 above)
Comment (dmytrotkk) [reply src/components/SFuelBadge/SFuelBadge.tsx]: fixed

File: src/components/TokenList/TokenList.scss (hunk 1 above)
Comment (dmytrotkk) [reply src/components/TokenList/TokenList.scss:6]: addressed

**Review by cstrangedk — APPROVED** [review]

**Review by TheGreatAxios — APPROVED** [review]

**Pull request merged**